pkg/sdk/go/sdk.go

```diff
@@ -408,3 +410,20 @@ func (pm PageMetadata) query() (string, error) {
 	}
 	return q.Encode(), nil
 }
+
+func encodeError(body []byte, status ...int) error {
```

Comment:
Why does it accept multiple statuses?

Reply:
I initially created errors with only the response body, and not the status code. So this is just so we can encode error with or without using http status. I'll change it back to using just one status.